src/js/facility-locator/components/SearchControls.jsx

```diff
@@ -48,6 +70,16 @@ class SearchControls extends Component {
 
   render() {
     const { currentQuery, isMobile } = this.props;
+    const { facilityDropdownActive } = this.state;
+
+    /* eslint-disable camelcase */
+    const facilityTypes = {
+      all: 'All',
+      va_health_facility: 'Health',
```

Comment:
@NatalieEMoore or @gnakm  -- What are the full names for these facilities? Should they say "Medical Centers", "Benefits Office" and "Cemetery"?